src/main/java/org/jabref/logic/integrity/PersonNamesChecker.java

```diff
@@ -15,6 +16,16 @@
         } else if (valueTrimmedAndLowerCase.endsWith(" and") || valueTrimmedAndLowerCase.endsWith(",")) {
             return Optional.of(Localization.lang("should end with a name"));
         }
+
+        // Check that the value is in one of the two standard BibTeX formats:
+        //  Last, First and ...
+        //  First Last and ...
+        AuthorList authorList = AuthorList.parse(value);
+        if (!authorList.getAsLastFirstNamesWithAnd(false).equals(value)
+                && !authorList.getAsFirstLastNamesWithAnd().equals(value)) {
+            return Optional.of(Localization.lang("Names are not in the standard BibTeX format."));
```

Comment:
Should we drop a hint here what the expected format actually is? Otherwise, the user might be left wondering what he should do to fix the warning.